wallet/build.gradle

```diff
@@ -120,4 +121,4 @@ dependencies {
 
     testImplementation libs.bundles.unit.testing
     testRuntimeOnly libs.junit.jupiter.engine
-}
\ No newline at end of file
+}
```

Comment:
linted or accidental?

Reply:
Automatically done by something (maybe Android Studio). Preferable this way, I think.
It'd hard to tell from the diff, but there's a newline here now, where I think there didn't used to be one.